fern/apis/docs-yml/definition/docs.yml

```diff
@@ -731,6 +731,7 @@ types:
       go: optional<SnippetLanguageConfiguration>
       java: optional<SnippetLanguageConfiguration>
       ruby: optional<SnippetLanguageConfiguration>
+      csharp: optional<SnippetLanguageConfiguration>
```

Comment:
https://github.com/fern-api/fern/blob/e778edf22a7a050cf2d20716a4c23d6a840de515/packages/cli/configuration-loader/src/docs-yml/parseDocsConfiguration.ts#L859-L872 guessing we will need some plumbing